tests/beaker/tests/create_vg_param_createonly.rb

```diff
@@ -0,0 +1,47 @@
+require 'master_manipulator'
+require 'lvm_helper'
+require 'securerandom'
+
+test_name "FM-4579 - C96596 - create volume group with parameter 'createonly'"
```

Comment:
The 'createonly' parameter implies some type of modification protection. No where in this test do you test this state of the volume group.